### PR TITLE
[llfio] Fix commit reference

### PR DESCRIPTION
**Pull request by BurningEnlightenment**
Due to ned14/llfio#115 the release had to be retagged. I suspect that the currently referenced commit will be garbage collected sooner or later whence the commit reference update.

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/llfio/portfile.cmake [hunk 1]
```diff
@@ -8,8 +8,8 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ned14/llfio
-    REF 62536e8dcb7108e2d355e109fe4a140e416ae16b
-    SHA512 b0a8ca833f9414a09c806468cc5961d8068c1008396172c455781e192cc2b9ff2a4c01c43804f511c17bcf7ff1bc426c2f95467a4647f44fc3cf81a455d734a5
+    REF b588deb7593d62e3afc025f85477d09b8012fb90
+    SHA512 47b43c716406d2847b16c33730376dc2a98ad8d079319387087ba4b37fdf5623c660fc53c92e2f90995a877001a7a739985d15d0b5133e079254e2d85ba5d8e4
```
Comment (LilyWangLL) [inline ports/llfio/portfile.cmake:12]:
```suggestion
    REF "${VERSION}"
    SHA512 06c5bdd6594a6fb9f69e2d93bf829edb56d2dd948120ad39435c15b7b7f64a623a5388ce19427d1413445df3c36df935269ca8441e775e0d9b61370c46f8c2f3
```

File: ports/llfio/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "llfio",
   "version-date": "2023-03-13",
+  "port-version": 1,
```
Comment (LilyWangLL) [inline ports/llfio/vcpkg.json:4]:
```suggestion
  "version": "20230311",
```

Comment (LilyWangLL) [reply ports/llfio/vcpkg.json:4]: I saw the upstream release 20230311, maybe could we use version 20230311 instead of commit ID?

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Comment by BurningEnlightenment** [comment]
@LilyWangLL the project follows a versioning policy aligned with `version-date`. The tags themselves exist for searchability and play an advisory role.

**Comment by LilyWangLL** [comment]
> @LilyWangLL the project follows a versioning policy aligned with `version-date`. The tags themselves exist for searchability and play an advisory role.

I got it, thanks for your reply.

**Review by LilyWangLL — APPROVED** [review]

**Pull request merged**